meta-oe/recipes-core/sdbus-c++/sdbus-c++_0.7.2.bb

```diff
@@ -0,0 +1,46 @@
+SUMMARY = "sdbus-c++"
+DESCRIPTION = "High-level C++ D-Bus library designed to provide easy-to-use yet powerful API in modern C++"
+
+SECTION = "libs"
+
+LICENSE = "LGPLv2.1"
+LIC_FILES_CHKSUM = "file://${WORKDIR}/sdbus-cpp-${PV}/COPYING;md5=1803fa9c2c3ce8cb06b4861d75310742"
```

Comment:
You dont need `${WORKDIR}/sdbus-cpp-${PV}/` here

Reply:
Fixed.